umh-core/examples/example-config-empty.yaml

```
@@ -20,3 +20,5 @@ internal:
   redpanda:
     desiredState: stopped
   services: []
+  redpanda:
```

YAMLlint (1.35.1)

[error] duplication of key "redpanda" in mapping

(key-duplicates)